README.md

```diff
@@ -24,16 +24,44 @@
 [conduct-shield]: https://img.shields.io/badge/%E2%9D%A4-code%20of%20conduct-%23f15a24.svg
 [conduct-link]: https://github.com/coq-community/manifesto/blob/master/CODE_OF_CONDUCT.md
 
-This GitHub action can be used together with
-[coqorg/coq](https://hub.docker.com/r/coqorg/coq/) Docker images.
+This is a GitHub action that uses (by default) 
+[coqorg/coq](https://hub.docker.com/r/coqorg/coq/) Docker images,
+which in turn is based on [coqorg/base](https://hub.docker.com/r/coqorg/coq/),
+a Docker image with a Debian environment.
+
+|   | GitHub repo       | Type          | Docker Hub
+|---|-------------------|---------------|-------------
+| x | docker-coq-action | GitHub action | <n/a>
+| ↳ | docker-coq        | Dockerfile    | coqorg/coq
+| ↳ | docker-base       | Dockerfile    | coqorg/base
+| ↳ | debian docker     | Docker image  | \_/debian
```

Comment:
For uniformity, I'd remove the `docker` keyword here (albeit it isn't a GH repo, but the `Type` column precises it anyway):
```suggestion
| ↳ | Debian            | Docker image  | \_/debian
```
```

Reply:
Well the repo as far as i could tell is docker-library/official-images
https://github.com/docker-library/official-images/blob/master/library/debian